all-a-board/src/components/Main/ReviewPage.jsx

```diff
@@ -1,23 +1,31 @@
 import './ReviewPage.css';
 import ReviewCard from "./ReviewCard";
 import VoteBox from './VoteBox';
+import Comment from './Comment';
 import { useParams } from 'react-router-dom';
 import { useEffect, useState } from 'react';
-import { getReview } from '../../axios';
+import { getReview, getComments } from '../../axios';
 
 export default function ReviewPage() {
     const [review, setReview] = useState()
+    const [showComments, setShowComments] = useState(false)
+    const [comments, setComments] = useState([])
     const [isErr, setIsErr] = useState(false)
     const {review_id} = useParams()
 
     useEffect(() => {
         getReview(review_id)
```

Comment:
There is no isLoading on any of your pages, I noticed it first on the categories drop down menu but its also missing here, you are only fetching the comments once, which means that you'd need to refresh in order toes for updated ones, You could extract this into its own comments component and do a new get request each time.